test/tests/problems/eigen_problem/initial_condition/gold/ne_ic_no_free_out_eigenvalues_0001.csv

```diff
@@ -1,2 +1,2 @@
 eigen_values_imag,eigen_values_real
-0,0.19994161438723
+0,1.0948556405555
```

Comment:
Explanation would be helpful whenever we are changing gold files.

Reply:
I can modify the message to include the reason, which has been explained in the issue.